src/pint/toa.py

```diff
@@ -73,6 +73,16 @@
 
 all_planets = ("jupiter", "saturn", "venus", "uranus", "neptune", "earth")
 
+tempo_aliases = {
+    "arecibo": "AO",
+    "jodrell": "JB",
+    "A1DOT": "XDOT",
+    "STIGMA": "VARSIGMA",
+    "EFAC": "T2EFAC",
+    "EQUAD": "T2EQUAD",
```

Comment:
This is from memory, but I'm pretty sure this is correct, but T2EFAC and T2EQUAD aren't really aliases for EFAC and EQUAD, right?  Their order of ops is different.  I'll see if I can't find the documentation for that....

Reply:
Hmmm.  After looking at the TEMPO and T2 codes, maybe I'm wrong and they both are implemented the same.  I swear this was a problem sometime in the past, but maybe it was an isolated incident or something.